benchpress/install_nginx.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -e
+set -x
+
+NGINX_VERSION='nginx-1.12.2'
+NGINX_TAR_FILE="${NGINX_VERSION}.tar.gz"
+NGINX_DOWNLOAD_URL="http://nginx.org/download/${NGINX_TAR_FILE}"
```

Comment:
Can we install a prebuilt version of nginx using yum? I see a few nginx versions available for CentOS, eg - https://www.cyberciti.biz/faq/how-to-install-and-use-nginx-on-centos-7-rhel-7/

Reply:
Sure, we can do that.

But the reasons I went with building from source are:
- No need for root privileges to install or run the benchmark
- Pinned version of the source code so we can reproduce the results.